tuned/plugins/plugin_scheduler.py

```diff
@@ -640,6 +640,13 @@ def _verify_irq_affinity(self, irq_description, correct_affinity,
 			log.info(consts.STR_VERIFY_PROFILE_VALUE_OK
 					% (irq_description, current_affinity))
 		else:
+			if not self._set_irq_affinity(self, irq, current_affinity, False):
```

Comment:
I don't think making changes to the system during verification is desirable. Also, I'm afraid it could potentially lead to security issues, as verification can be performed by any user on the system (based on our polkit policy).